src/layman/map/prime_db_schema/tasks.py

```diff
@@ -26,8 +26,11 @@ def refresh_bbox(
 
     mapjson = util.get_map_file_json(workspace, mapname)
     bbox_4326 = util.get_bbox_from_json(mapjson)
-    bbox_3857 = bbox_util.transform(bbox_4326, 4326, 3857)
-    set_bbox(workspace, MAP_TYPE, mapname, bbox_3857, )
+    crs = util.get_crs_from_json(mapjson)
+    assert crs == 3857
+    bbox = bbox_util.transform(bbox_4326, 4326, crs)
```

Comment:
transform() method should accept CRS as string, not integer

Vytáhnout `EPSG:4326` do konstanty